### ISSUE TITLE
Allow for a way to disable the automated calculations derived from levels of exhaustion

### ISSUE THREAD

**Issue by krbz999** [foundryvtt/dnd5e#2795]

**Comment by arbron** [comment]
Probably makes sense to have a world setting to disable all of the new automatic status effect calculations.

**Comment by krbz999** [comment]
Kim and I discussed this briefly, and I feel quite strongly that a config key is the way to go.

**Comment by Fyorl** [comment]
They are not mutually exclusive. I think a world setting to disable condition automation makes sense, similar to the various other settings we have that disable various automations. A config key to turn off specific condition automations is fine too, so long as it's not very complex to implement (I don't see why it would be though).

**Comment by JPMeehan** [comment]
Yeah the new Exhaustion handling - both from a UI as well as data perspective - seems to have voided my attempts to use the playtest Exhausted rules (even though those rules are no longer going to be in the revised core books I like them anyways)

**Comment by brucetony** [comment]
Would also like to be able to disable PHB exhaustion rules from being automatically applied. I want to use to the playtest rules as well, but not double penalize my players with PHB penalities

**Comment by JPMeehan** [comment]
> Would also like to be able to disable PHB exhaustion rules from being automatically applied. I want to use to the playtest rules as well, but not double penalize my players with PHB penalities

This is now doable with a world script and should probably be closed

**Comment by krbz999** [comment]
> This is now doable with a world script and should probably be closed

Disagree. There is a benefit in including a setting in the UI without the need for a world script.

**Comment by arbron** [comment]
### From [maxobremer](https://github.com/maxobremer):
With https://github.com/foundryvtt/dnd5e/issues/3920 exhaustion was implemented for the modern/2024 rules. It was also automated, applying penalties and the like.
It would be nice if this automation can be turned of for those that either dont want exhaustion, want to homebrew it or have a different reason. A simple button in the settings would do, maybe do the same for the auto attack roll? I think its a good idea, but some people might want more control over such options

**Comment by roth-michael** [comment]
Commenting for assignment
